addons/website_sale/data/data.xml

```diff
@@ -64,7 +64,7 @@
             <field name="model_id">product.product</field>
             <field name="user_id" eval="False" />
             <field name="domain">[('website_published', '=', True)]</field>
-            <field name="context">{'display_default_code': False}</field>
+            <field name="context">{'display_default_code': False, 'add2cart_rerender': False}</field>
```

Comment:
Commit message mention the wrong apps being modified

Reply:
Sorry about that 😓 it should be fixed now.